metrics/prometheus.go

```diff
@@ -1746,6 +1751,54 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 		})
 	}
 
+	if includedMetrics.Has(container.PressureMetrics) {
+		c.containerMetrics = append(c.containerMetrics, []containerMetric{
+			{
+				name:      "container_pressure_cpu_stalled_seconds_total",
```

Comment:
I'm trying to reason about which metrics make sense to emit. It's interesting we emit the totals but don't emit the 10/60/300... I almost wonder if we should add the structure pieces first and discuss actual metrics we emit after..

Reply:
No, we only need the totals, as other metrics are derived values. With the totals the end user can derive arbitrary intervals. For example `rate(container_pressure_cpu_stalled_seconds_total[60s])`.

Reply:
fair, I am not sure that prom query would get exactly the same but true that you could reconstruct those intervals from the total.

Reply:
Yea, it's never going to be exactly the same because it depends on exactly the timestamps and values involved.

But it will be within the same tolerance over time.